### PR TITLE
deployment files

### PR DESCRIPTION
**Pull request by LilianMathu**
@moshonk

Please review the deployment files for release one and offer feedback.

### CODE REVIEW

File: docker/docker-compose.test-apache.yaml [hunk 1]
```diff
@@ -23,9 +23,9 @@ services:
       dockerfile: Dockerfile.churchcrm-apache-test-debian
     #image: churchcrm/crm:php7.3-debian
     ports:
-      - "8081:80"
+      - "80:80"
     volumes:
-      - /opt/kcc:/var/www/html
+      - /home/lmathu/kcc:/var/www/html
```
Comment (moshonk) [inline docker/docker-compose.test-apache.yaml:28]: Put the project files in the `/opt/` directory

Comment (LilianMathu) [reply docker/docker-compose.test-apache.yaml:28]: @moshonk 
Okay.

**Review by moshonk — CHANGES_REQUESTED** [review]
The deployment location should  the `/opt`` directory

**Comment by LilianMathu** [comment]
@moshonk Hi Ken. I have made the changes you requested. Please review and revert.

**Comment by LilianMathu** [comment]
@moshonk Please review and offer feedback.

**Pull request closed without merging**